api/tests/acceptance/application/certification-courses/index_test.js

```diff
@@ -0,0 +1,309 @@
+                ],
+              },
+              {
+                id: 'recCompetence4',
+                tubes: [
+                  {
+                    id: 'recTube4_0',
+                    skills: [
+                      {
+                        id: 'recSkill4_0',
+                        nom: '@recSkill4_0',
+                        challenges: [{ id: 'recChallenge4_0_0' }],
+                        level: 0,
+                      },
+                      {
+                        id: 'recSkill4_1',
+                        nom: '@recSkill4_1',
+                        challenges: [{ id: 'recChallenge4_1_0' }],
+                        level: 1,
+                      },
+                      {
+                        id: 'recSkill4_2',
+                        nom: '@recSkill4_2',
+                        challenges: [{ id: 'recChallenge4_2_0' }],
+                        level: 2,
+                      },
+                    ],
+                  },
+                ],
+              },
+              {
+                id: 'recCompetence5',
+                tubes: [
+                  {
+                    id: 'recTube0_0',
+                    skills: [
+                      {
+                        id: 'recSkill5_0',
+                        nom: '@recSkill5_0',
+                        challenges: [
+                          { id: 'recChallenge5_0_0', langues: ['Franco Français'] },
+                          { id: 'recChallenge5_0_1' },
+                        ],
+                        level: 0,
+                      },
+                      {
+                        id: 'recSkill5_1',
+                        nom: '@recSkill5_1',
+                        challenges: [{ id: 'recChallenge5_1_1', langues: ['Franco Français'] }],
+                        level: 1,
+                      },
+                    ],
+                  },
+                ],
+              },
+              {
+                id: 'recCompetence6',
+                tubes: [
+                  {
+                    id: 'recTube0_0',
+                    skills: [
+                      {
+                        id: 'recSkill6_0',
+                        nom: '@recSkill6_0',
+                        challenges: [{ id: 'recChallenge6_0_0', langues: ['Anglais'] }],
+                        level: 0,
+                      },
+                      {
+                        id: 'recSkill6_1',
+                        nom: '@recSkill6_1',
+                        challenges: [{ id: 'recChallenge6_1_0', langues: ['Anglais'] }],
+                        level: 1,
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        ];
+        const learningContentObjects = learningContentBuilder.fromAreas(learningContent);
+        mockLearningContent(learningContentObjects);
+      });
+      it('should create it in database', async function () {
```

Comment:
Chipotage :
"should create the certification course in database" 😇

Reply:
Corrigé